src/Components/InputForm.test.tsx

```diff
@@ -0,0 +1,17 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import InputForm from './InputForm';
+
+test('InputForm renders correctly', () => {
```

Comment:
add a describe block to wrap all the tests inside. Look into the CTP test case files

Reply:
Fine I'll make the required changes.